core/events/workspace_comment/create.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
I'm also working on events right now.  Your code structure is similar to mine, though I created a base class for comment events.  I'm going to continue working on the other event types, and i'll handle any merge conflicts that result.

Reply:
Sounds good :)

Reply:
Looking at this again, I think it's best if you remove this file + references from this PR.  I'll be landing #1723 tomorrow, barring surprises.  No point in landing two versions of the code and then deleting one.

Reply:
Makes sense. I'll get rid of it.

Reply:
Done. Added a placeholder TODO to fire the event.